.gitignore

```diff
@@ -18,7 +18,7 @@
 /doc/*.tex
 /doc/*.toc
 /doc/manual.pdf
-/doc/NormalizInterface-bib.xml.bib
```

Comment:
There is more Normaliz specific stuff below (including the /test-suite.log and more)